cfme/tests/infrastructure/test_host.py

```diff
@@ -468,6 +468,43 @@ def test_compare_hosts_from_provider_allhosts(appliance, setup_provider_min_host
     assert compare_hosts_view.is_displayed
 
 
+@test_requirements.infra_hosts
+@pytest.mark.meta(blockers=[BZ(1747545, forced_streams=["5.10"])], automates=[1747545])
+@pytest.mark.meta(blockers=[BZ(1746214, forced_streams=["5.10"])], automates=[1746214])
+@pytest.mark.parametrize("num_hosts", [2, 4])
+@pytest.mark.parametrize("hosts_collection", ["provider", "appliance"])
+@pytest.mark.parametrize('report_format', IPAppliance.DownloadOptions, ids=[fmt.name for fmt in
+                                                                IPAppliance.DownloadOptions])
+def test_infrastructure_hosts_navigation_after_download_from_compare(
+        appliance, setup_provider_min_hosts, provider, report_format, hosts_collection, num_hosts
+):
+    """
+    Polarion:
+        assignee: prichard
+        casecomponent: Infra
+        caseimportance: high
+        initialEstimate: 1/3h
+    Bugzilla:
+        1747545
+
+    """
+    ent_slice = slice(0, num_hosts, None)
+    if hosts_collection == "provider":
+        hosts_view = navigate_to(provider.collections.hosts, "All")
+    elif hosts_collection == "appliance":
+        hosts_view = navigate_to(appliance.collections.hosts, "All")
+    for h in hosts_view.entities.get_all(slice=ent_slice):
+        h.ensure_checked()
+    hosts_view.toolbar.configuration.item_select('Compare Selected items',
+                                                 handle_alert=True)
+    hosts_view.toolbar.download.item_select(report_format.value)
+    if report_format.value == "Print or export as PDF":
```

Comment:
Why compare the value in this test case, and the name in the other?

Its best to actually compare the enum itself.

```
n [2]: for opt in IPAppliance.DownloadOptions: 
   ...:     print(f'Option: {opt}') 
   ...:     if opt == IPAppliance.DownloadOptions.PDF: 
   ...:         print('PDF match') 
   ...:                                                                                                                       
Option: DownloadOptions.TEXT
Option: DownloadOptions.CSV
Option: DownloadOptions.PDF
PDF match

```

Reply:
Updated as suggested.